frontend/src/main/web/src/components/View.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { PropTypes } from 'react' // eslint-disable-line
+import { flattenThemeClasses } from '../utils/styleUtils'
+
+const classes = {
+  base: {
+    ai: 'Ai(st)',
+    d: 'D(f)',
+    fld: 'Fld(c)',
+    flxs: 'Flxs(0)'
+  }
+}
+
+const View = ({
```

Comment:
Needs doc. What is a `View`? Is it like a panel or something? Looks like it is basically just a styled div.